make/envoy-gateway.mk

```diff
@@ -39,10 +39,19 @@ $(EGCTL):
 .PHONY: egctl
 egctl: $(EGCTL) ## Download egctl locally if necessary.
 
+envoy-gateway-enable-envoypatchpolicy: $(YQ)
```

Comment:
This is interesting, I guess if we document support for envoygateway in the repo / on the website we should call out this requirement to enable the envoy patch policy API

Reply:
Sure!